src/hotspot/share/gc/shared/vmStructs_gc.hpp

```diff
@@ -176,7 +171,6 @@
   declare_toplevel_type(CollectedHeap*)                                   \
   declare_toplevel_type(ContiguousSpace*)                                 \
   declare_toplevel_type(DefNewGeneration*)                                \
-  declare_toplevel_type(GenCollectedHeap*)                                \
   declare_toplevel_type(Generation*)                                      \
```

Comment:
Can these two *Generation be removed also? (They are specific to Serial, while this file is for "shared".)